### PR TITLE
Boostrap script

### PR DESCRIPTION
**Pull request by RHDZMOTA**
#### What does this PR do?
Creates a bootstrap bash script to automate the configuration procedure and setup.

#### Where should the reviewer start?
File in the base-path: `bootstrap.sh`

#### How should this be manually tested?
Follow the **fist usage** guide. Run the app: `python main.py`.

#### Any background context you want to provide?
* We hesitate about implementing type-annotation. 
* We are not sure about the best-practices for logging.

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+VENV_DIR="venv"
+if [ ! -d "$VENV_DIR" ]; then
+    virtualenv --python=python3 venv
+fi
+
+# Activate venv and install requirements
+source venv/bin/activate
+pip install -r requirements.txt
```
Comment (ocampor) [inline bootstrap.sh]: `requirements.txt` is hard-coded.

File: bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# create .env vars and run setup.
+cp settings/.env.example settings/.env
```
Comment (ocampor) [inline bootstrap.sh]: Don't do this by the user. You should include this step in the readme, because the user needs to set.

**Review by ocampor — CHANGES_REQUESTED** [review]

File: bootstrap.sh (hunk 2 above)
Comment (RHDZMOTA) [reply bootstrap.sh]: Given the changes proposed by @ocampor I think we can delete the file **bootstrap.sh** all along. Do you agree @joseluisvaz ?

Comment (joseluisvaz) [reply bootstrap.sh]: Ja

**Comment by ocampor** [comment]
Could you please close this PR, it is outdated @RHDZMOTA

**Pull request closed without merging**